feat: Add Terra SRPM and appstream page, remove Terra Policy page and move the contents into contributing, and new guidelines page. Also fix some stuff (#196)
FyraLabs/devdocs: pages/andaman/configuration.mdx

## Configuration

> Andaman uses the HashiCorp Configuration Language (HCL) for its configuration format, known for being used in software by HashiCorp such as Terraform, Consul, Vault, and Packer. Its syntax is similar to UCL and NGINX's configuration format.

Reviewer: ```suggestion
Andaman uses the HashiCorp Configuration Language (HCL) for its configuration format, you may be familiar with it if you've used Vault. Its syntax is similar to UCL and NGINX's configuration format.
```
we don't need to list a bunch of tools, pick the most popular

Author: Yes I did

> `{ }` represents a dictionary (empty by default) with keys and string values. `""` represents a string with no default value. Unless otherwise stated, all the attributes are optional. Consider the following code block.

Reviewer: ```suggestion
`{ }` represents a dictionary (empty by default) with keys and string values. `""` represents a string with no default value. Unless otherwise stated, all the attributes are optional. Here's an example:
```
